listenbrainz/webserver/static/js/src/stats/UserEntity.tsx

```diff
@@ -0,0 +1,449 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import * as ReactDOM from "react-dom";
+import * as React from "react";
+
+import APIService from "../APIService";
+import Bar from "./Bar";
+import ErrorBoundary from "../ErrorBoundary";
+
+export type UserEntityData = Array<{
+  id: string;
+  count: number;
+}>;
+
+export type UserEntityProps = {
+  user: ListenBrainzUser;
+  apiUrl: string;
+};
+
+export type UserEntityState = {
+  data: UserEntityData;
+  range: UserEntityAPIRange;
+  entity: Entity;
+  currPage: number;
+  entityCount: number;
+  totalPages: number;
+  maxListens: number;
+  startDate: Date;
+};
+
+export default class UserEntity extends React.Component<
```

Comment:
Should we call this `UserEntityGraph` or similar? `UserEntity` is a bit unclear to me.

Reply:
Changed to `UserHistory`